### PR TITLE
Add theorems on compact-sets-are-finite spaces

### LINKED ISSUE #179: Flesh out P136
https://topology.pi-base.org/properties/P000136 - compact subsets are finite

First we should add the alias "anticompact" used here: https://math.stackexchange.com/questions/517411/in-a-topological-space-only-finite-subsets-are-compact-sets

That source also suggests two theorems that will propagate the property in pi-base: every discrete space is p136, and T1 first countable p136 spaces are discrete.

We can also add every finite space is p136. I don't relish having finite spaces being both "compact" and "anticompact" so I don't suggest we change the canonical name for p136 in pi base, barring a lot of literature on the subject using the term I'm unfamiliar with.

I'll tackle this when I'm at a proper computer later this week.

### PR DESCRIPTION
**Pull request by StevenClontz**
Closes #179

### CODE REVIEW

**Comment by prabau** [comment]
As mentioned in a comment at the bottom of https://math.stackexchange.com/questions/517411, the proposed T292 can be generalized to: anticompact (P136) + Sequential (P79) + T1 (P2) ==> Discrete (P52).

The proof uses the same ingredients.  For example, given a point $x$ in an anticompact sequential T1 space, the complement $X\setminus\\{x\\}$ is sequentially closed (otherwise there would be a sequence $(x_n)$ converging to $x$ with each $x_n\ne x$.\; by the $T_1$ property, such a sequence must contain infinitely many different points, so the set of all those points together with $x$ would form a compact infinite set, contradicting the anticompactness property.)  Then by the sequential property, $X\setminus\\{x\\}$ is closed, and $x$ is an isolated point.

(I am starting to like the terminology "anticompact", short and sweet)

**Comment by prabau** [comment]
Just debating with myself whether anticompact is widely used or not.
https://scholar.google.com/scholar?start=10&q=anticompact&hl=en&as_sdt=0,33
shows a few topology papers using that term.  It's also used in other areas, differential equations, functional analysis, with completely different meanings.

Even on mo in topology, some people just make up stuff with complete disregard to what has been published before: https://mathoverflow.net/questions/191266/anti-compactness

**Comment by prabau** [comment]
For P000136.md, would it be possible to add a reference to the paper by Bankston, "The total negation of a topological property" (doi:10.1215/ijm/1256048236) to explain the origin of the name "anticompact" ?

File: theorems/T000294.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+uid: T000294
+if:
+  P000052: true
+then:
+  P000136: true
+---
+
+Infinite subsets are {P52} and therefore not {P16}.
```
Comment (prabau) [inline theorems/T000294.md:9]: I am confused by this sentence.  Maybe you meant something like "Infinite subsets of a discrete space are not compact".

File: theorems/T000292.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+if:
+  and:
+    - P000002: true
+    - P000028: true
+    - P000136: true
+then:
+  P000052: true
+refs:
+  - mathse: 517411
+    name: in a topological space only finite subsets are compact sets
```
Comment (prabau) [inline theorems/T000292.md:12]: I changed the title of the mathse post to the more readable "Is a space where only finite subsets are compact sets always discrete?" [here](https://math.stackexchange.com/questions/517411).

**Review by prabau — APPROVED** [review]

**Comment by StevenClontz** [comment]
Thanks for the approval and great feedback. Sorry for the contextless edits towards the end - folks have been coming in and out of the office today and I've been tweaking in between meetings. :)

**Pull request merged**